src/Microsoft.Azure.WebJobs.Host/Executors/IMethodInvoker.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Azure.WebJobs.Host.Executors
 {
-    internal interface IMethodInvoker<TReflected, TReturnValue>
+    public interface IMethodInvoker<TReflected, TReturnValue>
```

Comment:
Exposing the method invoker factory publically so Functions Host can use compiled lambdas rather than reflection to invoke methods, just like the SDK does